components/graphs/ThresholdGraph.tsx

```diff
@@ -31,14 +32,30 @@ const ThresholdGraph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
   const { forecastData, latitude, longitude, isLoading } =
     useSiteData(siteUUID);
   const { currentTime } = useTime(latitude, longitude);
-
+  let currentDate = new Date();
+  let endDate = new Date();
+  currentDate.setHours(8);
+  currentDate.setMinutes(0);
+  currentDate.setSeconds(0);
+  currentDate.setMilliseconds(0);
```

Comment:
Maybe instead of computing these in the components, something like this could work?
```
  // Defined somewhere above the component...
  function getGraphEndDate(date: Date) {
    endDate.setHours(20);
    endDate.setMinutes(0);
    endDate.setSeconds(0);
    endDate.setMilliseconds(0);
  }

  // In the component
  const endDate = getGraphEndData(new Date());
```